.github/workflows/linux-dispatch-build-bottle.yml

```diff
@@ -0,0 +1,143 @@
+name: Dispatch build bottle (Linux)
+
+on:
+  workflow_dispatch:
+    inputs:
+      formula:
+        description: Formula name
+        required: true
+      issue:
+        description: Issue number, where comment on failure would be posted
+        required: false
+      upload:
+        description: Whether to upload built bottles to Bintray or not
+        required: false
+
+env:
+  HOMEBREW_DEVELOPER: 1
+  HOMEBREW_GITHUB_ACTIONS: 1
+  HOMEBREW_NO_AUTO_UPDATE: 1
+
+jobs:
+  bottle:
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/ubuntu16.04:master
+    env:
+      HOMEBREW_GIT_NAME: BrewTestBot
+      HOMEBREW_GIT_EMAIL: 'homebrew-test-bot@lists.sfconservancy.org'
+      HOMEBREW_NO_BOTTLE_SOURCE_FALLBACK: 1
```

Comment:
This environment variable has been removed and no longer exists.

Reply:
It's still in the macOS dispatch build workflow though.

Reply:
Yeah it was removed a week or so ago and the workflows haven't been updated yet.

Reply:
I removed it from both workflows in this PR.